internal/pkg/event/types.go

```diff
@@ -0,0 +1,18 @@
+package event
```

Comment:
Better move this file into /internal/pkg/externalplugins.

Because only external plugins respond to these events.